ADO JS Mappings/ADO-ID-to-TP-URL

```diff
@@ -0,0 +1,55 @@
+Mapping of TP URL CF to ADO ID
+
+
+From TP to ADO
```

Comment:
You can add some structure for the document, for example transform that text to the header with `###`.
That will increase readability and allow us to send links for particular document part, for example: https://github.com/TargetProcess/JS-mapping-templates/blob/6c8473c11529cc337acc76624c122942cc268929/ADO%20JS%20Mappings/art-to-area-path.md#ado--tp